### PR TITLE
fix(react-kit): fixed touch events

### PR DESCRIPTION
**Pull request by green176**
Should fix [this bug](https://github.com/devexperts/dx-platform/issues/211)

### CODE REVIEW

File: packages/react-kit/src/components/EventListener/EventListener.tsx [hunk 1]
```diff
@@ -51,7 +51,8 @@ export class EventListener extends React.Component<TEventListenerProps> {
 			const capture = key.endsWith(CAPTURE_MARKER);
 			const handler = handlers[key];
 			const eventName = getEventName(key, capture);
-			target.addEventListener(eventName, handler as any, capture);
+			const passive = !['touchcancel', 'touchend', 'touchmove', 'touchstart'].includes(eventName);
```
Comment (raveclassic) [inline packages/react-kit/src/components/EventListener/EventListener.tsx]: are you sure you want to process _all_ events in passive mode?

File: packages/react-kit/src/components/RootClose/RootClose.tsx [hunk 2]
```diff
@@ -61,7 +61,8 @@ export class RootClose extends Component<TRootCloseProps> {
 		this.preventMouseRootClose = domNode.contains(e.target as Node);
 	};
 
-	private handleTouchStart: TouchEventHandler<HTMLElement> = () => {
+	private handleTouchStart: TouchEventHandler<HTMLElement> = e => {
+		e.preventDefault();
```
Comment (raveclassic) [inline packages/react-kit/src/components/RootClose/RootClose.tsx]: What exactly does it prevent?

Comment (green176) [reply packages/react-kit/src/components/RootClose/RootClose.tsx]: It should prevent click event

File: packages/react-kit/src/components/EventListener/EventListener.tsx (hunk 1 above)
Comment (green176) [reply packages/react-kit/src/components/EventListener/EventListener.tsx]: I [update](https://github.com/devexperts/dx-platform/pull/212/commits/6db8313000f284d4aef0c7b1fdef4127a722092a) it

**Comment by sutarmin** [comment]
Merged under #224

**Pull request closed without merging**